google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITBatchReadTest.java

```diff
@@ -181,7 +181,9 @@ public static void setUpDatabase() throws Exception {
           totalSize = 0;
         }
       }
-      dbClient.write(mutations);
+      if (mutations.size() > 0) {
```

Comment:
nit: do we really need this check? (and if so, could we change is to `!mutations.isEmpty()`)